scripts/vm/hypervisor/kvm/kvmheartbeat.sh

```diff
@@ -155,10 +155,10 @@ then
   exit 0
 elif [ "$cflag" == "1" ]
 then
-  /usr/bin/logger -t heartbeat "kvmheartbeat.sh rebooted system because it was unable to write the heartbeat to the storage."
+  /usr/bin/logger -t heartbeat "kvmheartbeat.sh stopped cloudstack-agent because it was unable to write the heartbeat to the storage."
   sync &
   sleep 5
-  echo b > /proc/sysrq-trigger
+  systemctl stop cloudstack-agent
```

Comment:
@Slair1 are you sure this will work with CentOS6 and debian KVM hosts? Same goes with the change above

Reply:
Good thought, I can change that to “service” for older version compatibility